docs/advanced-scenarios.md

```diff
@@ -205,4 +205,48 @@ Luigi allows you to implement and configure feature toggles. They can be used to
     }
   ``` 
 
+### Use Intent Based Navigation in Luigi Client
+
+#### Overview 
+Luigi Client allows you to navigate through microfrontends by using an intent based navigation.
+
+#### Usage
+* To **enable** intent based navigation, you need to first identify the necessary target mappings. This can be defined by defining `intentMapping` to the LuigiConfig under `navigation` as in the example below:
+  ```javascript
+  intentMapping = [
+    {
+      semanticObject: 'Sales',
+      action: 'display',
+      pathSegment: '/projects/sap/munich/database/sales/display'
+    },
+    {
+      semanticObject: 'Sales',
+      action: 'edit',
+      pathSegment: '/projects/sap/munich/database/sales/edit'
+    }
+  ];
+  ```
+  1. The intent link is built using the semanticObject, action and optional parameters in the following format:
+  `#?Intent=semanticObject-action?params`
+  An example of an intent link would be as follows:
+  ```javascript
+    #?Intent=Sales-edit?id=100
+  ```
+  2. Navigation to a microfrontend through this intent is then made possible by using the linkManager navigate method from [Luigi Client API](luigi-client-api.md#navigate):
+  ```javascript
+    LuigiClient.linkManager().navigate('#?Intent=Sales-edit?id=100');
+  ```
+
+  3. This would then be navigating to the translated real path segment:
```

Comment:
```suggestion
  3. This method would then be navigating to the translated real path segment:
```
The word "this" without referring to anything sounds a bit confusing, as far as I understand it refers to the linkManager navigate method mentioned above?